2-mandatory/3-stocks.js

```diff
@@ -65,9 +111,21 @@ function getPriceChanges(closingPricesForAllStocks) {
 */
 function highestPriceDescriptions(closingPricesForAllStocks, stocks) {
     // TODO
+  let result = [];
+
+  for (let i = 0; i < stocks.length; i++) {
+    let stockPrices = closingPricesForAllStocks[i];
+    let highestPrice = Math.max(...stockPrices).toFixed(2);
```

Comment:
Excellent use of the max function by feeding in an array using 3dot notation.

Reply:
This one I spent over 2 hours -_- on after Google so much I used 3 point notation